pkg/addons/data/ConfigMap/openshift-metrics/prometheus.yaml

```diff
@@ -24,57 +24,6 @@ data:
     # default named port `https`. This works for single API server deployments as
     # well as HA API server deployments.
     scrape_configs:
-    - job_name: 'kubernetes-apiservers'
```

Comment:
This should work.  I'm guessing it doesn't?  If so can you find out approximately why?

Reply:
Do we want to give access to HCP metrics for clients? As this would be client side Prometheus scraping HCP component?

Reply:
@jim-minter this does not work due certificate. By default endpoints point to local address. In our case EP points to IP `https://10.244.x.xxx:443/metrics`

And certificate is valid only for:
`Get https://10.244.x.xxx:443/metrics: x509: certificate is valid for 172.30.0.1, not 10.244.x.xxx` 

`kubernetes         10.244.x.xxx:8053,10.244.x.xxx:8053,10.244.x.xxx:443   28m`